<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pythonChart</class>
 <widget class="QWidget" name="pythonChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>23</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="midLineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>Python图表制作器</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_areaChart">
      <attribute name="title">
       <string>面积图</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>数据设置</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>图表展示</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>X轴范围：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>最小值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_minX"/>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>最大值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_maxX"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_setX">
              <property name="text">
               <string>设置</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Y轴范围：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>最小值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_minY"/>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>最大值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_maxY"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_setY">
              <property name="text">
               <string>设置</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>序列1：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>X值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_series1X"/>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Y值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_series1Y"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_setSeries1">
              <property name="text">
               <string>添加</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>序列2：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>X值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_series2X"/>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Y值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_series2Y"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_setSeries2">
              <property name="text">
               <string>添加</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>序列1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_series1">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>序列2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_series2">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0" colspan="2">
           <widget class="QChartView" name="graphicsView"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_clear">
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_barChart">
      <attribute name="title">
       <string>条形图</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QChartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
